app/assets/stylesheets/_screen.scss

```diff
@@ -625,6 +617,63 @@ div#keycut-help {
   }
 }
 
+/* Settings */
+.settings-page {
+  height: 100%;
+  padding: 0 30px;
+  overflow-y: auto;
+}
+
+/* Import */
+.import-form {
+  margin: 15px auto;
+  width: 70%;
+
+  .field {
+    &.select {
+      padding: 5px;
+      background: $lightgrey;
+      border-radius: 5px;
+      padding: 10px;
+      margin-bottom: 10px;
+    }
+    &.submit {
+      text-align: right;
+    }
+  }
+}
+
+.instructions > p {
+  font-size: 1em;
+  margin-bottom: 20px;
+}
+
+/* Integrations */
+.panel {
+  .panel-content {
+    padding: 15px;
+  }
+}
+
+.integration > .panel-heading {
+  padding: 10px 15px;
+  vertical-align: middle;
+  span {
```

Comment:
Prefer using a class here. Also inline elements (like span) can't have a custom height/width, this `height: 100%` probably doesn't work.